README.md

```diff
@@ -18,7 +18,21 @@ This module provides Stackdriver Trace support for Node.js applications. [Stackd
 1. Your application will need to be using Node.js version 4.0 or greater.
 1. You will need a project in the [Google Developers Console][cloud-console]. Your application can run anywhere, but the trace data is associated with a particular project.
 1. [Enable the Trace API](https://console.cloud.google.com/flows/enableapi?apiid=cloudtrace) for your project.
-1. At this point, code using untranspiled async/await is not supported by this module. See [this section](#how-does-automatic-tracing-work).
+
+### Using `async/await`
```

Comment:
Note that this might confuse people on whether the API of this module can be used with async/await rather than the ability of this module to trace code using async await...

Reply:
Title changed to "Tracing with `async/await`"